conman/src/scheme.cpp

```diff
@@ -1759,6 +1802,39 @@ bool Scheme::disableBlocks(
   return success;
 }
 
+bool Scheme::disableBlocksTopo(
+    const std::vector<std::string> &unordered, 
+    const bool strict)
+{
+  using namespace conman::graph;
+
+  //Create non const list of unordered blocks to disable so we can add new
+  //arbitrary last element and then use std::find
+  std::vector<std::string> block_names = unordered;
+  block_names.push_back("/0");
```

Comment:
This should be unnecessary. If an item isn't found in an stl container, it will return an interator pointing to the end of the of the container. This end isn't the last element, but a pointer positioned after the last element.